src/app/app.component.ts

```diff
@@ -56,12 +56,22 @@ export class AppComponent implements OnInit, OnDestroy {
       this.apiKey = params.get('sdkKey');
       this.baseUrl = params.get('baseUrl');
       this.featureFlagKey = params.get('featureFlagKey');
+      let hideControls = params.get('hideControls');
 
-      if (!this.featureFlagKey) { this.featureFlagKey = 'isAwesomeFeatureEnabled'; }
+      if (!this.featureFlagKey) { this.featureFlagKey = ''; }
       this.apiKeyFormGroup = this.formBuilder.group({ apiKey: [this.apiKey, Validators.required] });
       this.featureFlagKeyFormGroup = this.formBuilder.group({ featureFlagKey: ['', Validators.required] });
       this.userCountFormGroup = this.formBuilder.group({ userCount: [20, Validators.required] });
 
+      if(this.apiKey && this.apiKey.length > 0) {
```

Comment:
```suggestion
      if(this.apiKey) {
```

Reply:
It's because of `if (!this.featureFlagKey) { this.featureFlagKey = ''; }`